website/sampling-consensus.md

```diff
@@ -1,37 +1,79 @@
-# Atoma's Sampling Consensus
+##  Issue with Trust 
 
-In order to process requests, the Atoma Network protocol selects a given
-number of nodes uniformly at random. The selected nodes must execute the
-given requests, otherwise part of their collateral value is at risk
-of being slashed. Moreover, the selected nodes are guaranteed to have similar hardware and software specifications. The node sampling mechanism is part of our `Sampling Consensus` protocol. The latter allows the Atoma Network to satisfy the following features:
+In a decentralized network, you can’t just run an LLM and trust the output. If we ask the network to run inference using Llama2-70B, how can we be assured that it’s not actually using LLama2-13B? 
 
-1. Load balancing of the request volume across the Atoma Network. As requests are only executed across nodes with specific hardware capacities, this node selection homogeneizes the balancing of the request load on the Atoma Network.
+Verifiable inference exists so that models not only respond to prompts but prove that it ran correctly on the desired model. 
 
-2. The selection mechanism allows nodes to reach deterministic agreement on the state of the output generated by executing the given request. This means that all selected nodes will generate the same outputs, given the same inputs and the same workload (specified by the incoming request). This is specially important for workloads that rely on floating point arithmetic, the latter being highly non-deterministic. Such an important example regards AI inference, which depends crucially on floating point arithmetic (specially for LLM inference). However, if we select nodes with the same hardware specifications (such as, the same GPU card), determinism of floating point arithmetic is guaranteed (with potential mild changes to the kernels which the code runs into).
+It’s technically possible to run LLM inference as a smart contract on-chain, but it would incur huge costs. Therefore we have to look into alternative methods of verification. 
```

Comment:
I disagree, I don't think it is technically possible. Maximum transaction for blockchains is a few kbs each, these models require hundreds of gigabytes. So I would suggest rewriting to its infeasible to run LLM inference